src/com/sap/piper/PiperGoUtils.groovy

```diff
@@ -1,16 +1,19 @@
 package com.sap.piper
 
+import java.util.List
+
 class PiperGoUtils implements Serializable {
 
-    private static Script steps
+    private static String additionalConfigFolder = ".pipeline/additionalConfigs"
+    private static def steps
     private static Utils utils
 
-    PiperGoUtils(Script steps) {
+    PiperGoUtils(def steps) {
         this.steps = steps
         this.utils = new Utils()
     }
 
-    PiperGoUtils(Script steps, Utils utils) {
+    PiperGoUtils(def steps, Utils utils) {
```

Comment:
Why removing the type?

Reply:
In order keep testing simpler: In `PiperGoUtilsTest#testHandleResources` I use a simple map with some mocks for the steps. This does not work when we are type save here. Would be more complicated when doing this with our rules.

Testing always affect the appearance of the code under test. That here might be an edge case. In case it distracts you too much, I can try to refactor. Just let me know ...

Reply:
@daniel-kurzynski : I checked more details wrt removed type. We have other cases where we test utils with our rules. But in that case the utils are simple grooy scripts rather than classes (e.g. DesctiptorUtil). In this case things like `echo`, `readFile` etc works fine. But with a class we have to provide the handle for the scripts (aka `script`, `step`). In this case in turn our rules does not work. Hence I would like to suggest to go forward with the removed type.

Reply:
we do have `nullscript` for testing.
From my POV we should not change the type here

Reply:
Don't remember the details, that wasn't that easy. But don't remember the details exactly. I will revisit ...